source/_integrations/comfoconnect.markdown

```diff
@@ -73,10 +73,14 @@ sensor:
       - outside_humidity
       - outside_temperature
       - power_usage
+      - preheater_power_usage
```

Comment:
Something minor, but could you add the new items in order (alphabetically) to the list? This makes it easier to look at the list to know what sensors are available.